AppKit/CPWindow/CPWindow.j

```diff
@@ -1858,7 +1858,7 @@ CPTexturedBackgroundWindowMask
             case CPLeftMouseDown:
 
                 // This is needed when a doubleClick occurs when the sheet is closing or opening
-                if (!_parentWindow)
+                if (!_parentWindow && [anEvent clickCount] > 1)
```

Comment:
Scratch that. It seems that _parentWindow is related to the child window feature, not sheets.
Maybe there is a confusion with _parentView. Can you check that ? The original PR is #2021

Reply:
_parentWindow is also nil when you try to move the underlying window and the sheet is open. this is what causes issue #2567.

Reply:
So, do you think #2021 is correct ?

Reply:
no, #2021 introduced a more serious issue than the one they aimed to fix.
the issue that motivated #2021 seems to be confined to double-clicks, as they discuss in the comment.
this is why i added the clickCount >1 constraint.

Reply:
I prefer that we fix the original issue instead of fixing the fix :)
The problem in #2021 was that -attachedSheet was returning YES while the sheet was closing and we were acting on a sheet that was in fact no here anymore.

This works for me:
```
L 1854: if (sheet && _sheetContext["isAttached"])
```
I haven't checked if in cocoa -attachedSheet return YES like we do when the sheet is closing. if not, there would be the place to fix #2021.

Reply:
thank you. i updated the PR accordingly. With respect to #2021 i do not have a reduction of the underlying issue.
However, i am not seeing any issues when double clicking on disappearing sheets (with this PR).